### PR TITLE
Update installing-local-extensions.md

### PR DESCRIPTION
**Pull request by mkovel**
i improved some details in step 4

### CODE REVIEW

File: docs/howto/installing-local-extensions.md [hunk 1]
```diff
@@ -92,12 +92,19 @@ Step 4
...
+
+from: `{web_root}/extensions/local/{author_name}/{extension_name}/web`
+
+to: `{public_root}/extensions/local/{author_name}/{extension_name}/`
 
 Where:
+ - `{web_root}` is the install location of your Bolt site
  - `{public_root}` is the public folder of your Bolt site
- - `{author_name}` is a lower-case, space-less name
- - `{extension_name}` is a lower-case, space-less name
```
Comment (GawainLynch) [inline docs/howto/installing-local-extensions.md]: For <= 3.3 lower case is required

File: docs/howto/installing-local-extensions.md [hunk 2]
```diff
@@ -92,12 +92,19 @@ Step 4
 
 Bolt might access your public assets (javascripts, stylesheets, images, etc.).
 For local extension, copy your assets manually to :
-`/{public_root}/extensions/local/{author_name}/{extension_name}/`
+`{public_root}/extensions/local/{author_name}/{extension_name}/`
+
+or just create symlink
```
Comment (GawainLynch) [inline docs/howto/installing-local-extensions.md]: Symlinks are not supported by Flysystem, and if they work only do so by fluke

**Comment by mkovel** [comment]
@GawainLynch  Is that better?
Because i had problems with creation directory for local extension assets. 
The documentation says about lower case, but in 3.2.6 using lower-case in path for assets results in an error.

**Comment by GawainLynch** [comment]
> Because i had problems with creation directory for local extension assets.

Which was something you unfortunately did incorrectly with your install, hence why this method has been reworked in 3.3.

Prior to 3.3, somethings may work out of luck with mixed case, but there are some comparisons done on the Composer name.

File: docs/howto/installing-local-extensions.md [hunk 3]
```diff
@@ -92,12 +92,19 @@ Step 4
 
 Bolt might access your public assets (javascripts, stylesheets, images, etc.).
 For local extension, copy your assets manually to :
-`/{public_root}/extensions/local/{author_name}/{extension_name}/`
+`{public_root}/extensions/local/{author_name}/{extension_name}/`
+
+or just create symlink ( not guaranteed )
```
Comment (GawainLynch) [inline docs/howto/installing-local-extensions.md]: Not supported means we won't add them to the documentation, as that implys we do.

**Comment by mkovel** [comment]
I did only what is written in the documentation https://docs.bolt.cm/3.2/howto/installing-local-extensions

1. I created a local extension in `extensions/local/Test/NewAdminMenu` 
2. I put "clippy.css" in the directory `extensions/local/Test/NewAdminMenu/web`
3. wrote in  'extensions/local/DK/NewAdminMenu/src/NewAdminMenuExtension.php:registerAssets ()'
` return [
            (new Stylesheet('clippy.css'))->setZone(Zone::BACKEND)
        ];
`
4. copy files from `extensions/local/DK/NewAdminMenu/web` to `public/extensions/local/dk/newadminmenu` **as says in docs**

and i had error **Couldn't add file asset 'clippy.css': File does not exist in either web://extensions/local/DK/NewAdminMenu or /theme/base-2016/ directories**

But in docs says :
 - `{author_name}` is a lower-case, space-less name
 - `{extension_name}` is a lower-case,  space-less name

I think if I have a problem with this, the others can be.

**Comment by GawainLynch** [comment]
In the [documentation here](https://docs.bolt.cm/3.2/howto/installing-local-extensions#step-1) it says:

> Create the directory for you extension in {web_root}/extensions/local/{author_name}/{extension_name}/
> 
> Where:
> 
> * {web_root} is the install location of your Bolt site
> * {author_name} is a lower-case, space-less name
> * {extension_name} is a lower-case, space-less name

Yet you specify that you put your extension in `extensions/local/Test/NewAdminMenu`.

Having written the code myself, I am pretty sure I know how it works :wink:

**Comment by mkovel** [comment]
@GawainLynch  
Aaaa !!! I understand you. :)  OK ok

**Comment by bobdenotter** [comment]
Just a quick note: We're aware that it is complicated and can be fickle and fragile. That's why we've refactored the "local extensions" in the upcoming Bolt 3.3. See here for the almost finished docs: 

https://github.com/bolt/docs/blob/release/3.3/docs/howto/installing-local-extensions.md

**Pull request closed without merging**